xoinvader/animation.py

```diff
@@ -64,10 +64,13 @@ def add(self, name, bind, attr, keyframes, loop=False):
     def update(self):
         """Update manager's state."""
 
+        if not self._animation:
+            return
```

Comment:
Dunno. Maybe add a flag.

Anyway, it's not used anywhere right now, but was quite a hindrance for me. So I have taken the liberty to remove the raising.

I personally don't have any consistent opinion on it yet, so it's up to you to decide, I'll obey any decision.